### PR TITLE
[#172184888] Make Ombulabs' Styleguide responsive again 

### PR DESCRIPTION
**Pull request by nykka**
Pivotal story reference: https://www.pivotaltracker.com/story/show/172357798
This PR: 

- Adds new styles to make Styleguide look better on mobile view

How it looks right now:
Screen Shot 2020-04-16 at 16 26 30

How it will look after this PR:
Screen Shot 2020-04-16 at 16 25 43

### CODE REVIEW

File: vendor/assets/stylesheets/ombulabs/styleguide/5-pages/_styleguide.scss [hunk 1]
```diff
@@ -40,6 +48,18 @@
     background-color: $black;
     font-size: 14px;
     padding-top: 40px;
+    @include respond-below(small) {
+      width: 100%;
+      right: -100vw;
+      left:auto;
+    }
+    &.open{
+	    right: 0;
```
Comment (arielj) [inline vendor/assets/stylesheets/ombulabs/styleguide/5-pages/_styleguide.scss:57]: indentation is wrong here

**Comment by ABizzinotto** [comment]
@nykka please ping me when you've addressed Ariel's comments and I can test this on staging :)

**Comment by nykka** [comment]
@arielj I couldn't replicate the issue, so I did what you suggested (added `overflow:hidden` to body when `modal-active`. Let me know if it works!

**Comment by arielj** [comment]
It's fixed now :+1: @nykka

**Comment by nykka** [comment]
@ABizzinotto approved by @arielj

**Pull request merged**